go.mod

```diff
@@ -125,3 +125,5 @@ require (
 	google.golang.org/protobuf v1.28.1 // indirect
 	gopkg.in/ini.v1 v1.67.0 // indirect
 )
+
+replace github.com/minio/madmin-go/v2 => github.com/poornas/madmin-go/v2 v2.0.0-20230406162832-ee76020dca2b
```

Comment:
please madmin-go to the newer release that you want @poornas `git tag` and `git push -u origin --tags`